test/s3_test_script.py

```diff
@@ -121,7 +120,7 @@ def create_iam_role(role_name):
 
     session = boto3.session.Session(profile_name='default')
     iam = session.client('iam')
-    response = iam.create_role(
```

Comment:
its always good to collect response, why removing? was it throwing warning that response var was unused. if yes, just print respone[status] like in other functions